modules/installing-gcp-user-defined-labels-and-tags.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+// * installing/installing_gcp/installing-gcp-customizations.adoc
+
+:_content-type: CONCEPT
+[id="installing-gcp-user-defined-labels-and-tags_{context}"]
+= User-defined labels and tags for GCP
```

Comment:
I know I suggested this. But looking at the topic hierarchy, I think we should rename to something like below:

```suggestion
= Managing user-defined labels and tags for GCP
```

Reply:
Ack.